src/api/queues.js

```diff
@@ -33,18 +33,11 @@ function validateLocation(req, res, next) {
 router.get(
   '/',
   safeAsync(async (req, res, _next) => {
-    const { userAuthz } = res.locals
-
-    const queuesResult = await Queue.scope('questionCount').findAll()
-    const queues = queuesResult.map(queueResult => {
-      const queue = queueResult.get({ plain: true })
-      if (isUserStudent(userAuthz, queue.courseId)) {
-        Queue.privateAttributes.forEach(attr => {
-          delete queue[attr]
-        })
-      }
-      return queue
-    })
+    const queuesResult = await Queue.scope(
+      'defaultScope',
```

Comment:
Is specifying `defaultScope` explicitly required?

Reply:
We need `defaultScope` so that the private attribute filtering gets applied, otherwise only `questionCount` will be used.